tests/rptest/tests/datalake/simple_connect_test.py

```diff
@@ -68,163 +62,45 @@ def __init__(self, test_context):
-        connect.start_stream(name="ducky_stream",
-                             config=self.avro_stream_config(
-                                 self.TOPIC_NAME, "verifier_schema", 3000))
+    @cluster(num_nodes=6)
+    @matrix(cloud_storage_type=supported_storage_types())
+    def test_translating_avro_serialized_records(self, cloud_storage_type):
+        connect = RedpandaConnectService(self.test_context, self.redpanda)
+        connect.start()
+        verifier = DatalakeVerifier(self.redpanda, self.TOPIC_NAME,
+                                    self.dl.spark())
+        mapping = dict(
+            ordinal="this",
+            timestamp="timestamp_unix_milli()",
+            verifier_string="uuid_v4()",
+        )
+        avro_stream_config = counter_stream_config(self.redpanda,
+                                                   self.TOPIC_NAME,
+                                                   "verifier_schema", mapping,
+                                                   3000)
+        connect.start_stream(name="ducky_stream", config=avro_stream_config)
         verifier.start()
         connect.stop_stream("ducky_stream")
         verifier.wait()
-
-    def verify_unmount_scenario(self, connect, verifier):
-        connect.start_stream(name="ducky_stream",
-                             config=self.avro_stream_config(
-                                 self.TOPIC_NAME, "verifier_schema", 1000000))
-        # todo make reasonable or just make sure something went through
-        self.redpanda.set_cluster_config({
-            "iceberg_catalog_commit_interval_ms":
-            self.SLOW_COMMIT_INTVL_S * 1000
-        })
-        verifier.start(wait_first_iceberg_msg=True)
-        self.admin = Admin(self.redpanda)
-        ns_topic = NamespacedTopic(self.TOPIC_NAME)
-
-        out_migration = OutboundDataMigration([ns_topic], consumer_groups=[])
-        out_migration_id = self.create_and_wait(out_migration)
-        self.admin.execute_data_migration_action(out_migration_id,
-                                                 MigrationAction.prepare)
-        self.wait_for_migration_states(out_migration_id, ['prepared'])
-        self.admin.execute_data_migration_action(out_migration_id,
-                                                 MigrationAction.execute)
-        # the topic goes read-only during this wait
-        self.wait_for_migration_states(out_migration_id, ['executed'])
-        connect.stop_stream("ducky_stream", should_finish=False)
-        time.sleep(1)  # just it case: let verifier consume remaining messages
-        verifier.go_offline()
-
-        self.admin.execute_data_migration_action(out_migration_id,
-                                                 MigrationAction.finish)
-        self.wait_for_migration_states(out_migration_id, ['finished'])
-        self.wait_partitions_disappear([self.TOPIC_NAME])
-
-        verifier.wait(progress_timeout_sec=10 * self.SLOW_COMMIT_INTVL_S)
```

Comment:
@bashtanov on my local machine, this test ran for 12 minutes. I don't believe this is a result of my changes here -- is this call to `verifier.wait(10min)` actually what you want? It appears to be waiting that long, and it doesn't throw on timeout.

Reply:
FWIW 12 minute runs seem to be common recently for this test https://buildkite.com/organizations/redpanda/analytics/suites/redpanda-1/tests?branch=dev&period=7days&query=simple_connect_test&commit=Search

Reply:
ping @bashtanov